src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -283,6 +290,38 @@ private void handleProjectileChangeBlock(EntityChangeBlockEvent event, Projectil
         event.setCancelled(true);
     }
 
+    private void handleEntityMeltPowderedSnow(@NotNull EntityChangeBlockEvent event)
+    {
+        // Note: this does not handle flaming arrows; they are handled earlier by #handleProjectileChangeBlock
+        Player player = null;
+        if (event.getEntity() instanceof Player localPlayer)
+        {
+            player = localPlayer;
+        }
+        else if (event.getEntity() instanceof Mob mob)
+        {
+            // Handle players leading packs of zombies.
+            if (mob.getTarget() instanceof Player localPlayer) player = localPlayer;
```

Comment:
Hmm, is convention sameline oneliners, or newline oneliners? Thought it was the latter, but I haven't been working with this codebase much lately (and I hopefully plan to change that within the coming weeks).

Reply:
A mix, usually one line for shorter ones. If you want to move to always having a newline, that's fine by me. Either way it feels a little alien, I always use braces these days, even for single lines, but with the newline before opening braces that takes up a lot of space.

Reply:
I always do newline for oneliners, since at least for my brain, I always recognize one line for performing either an action or evaluating a condition, not both.

Reply:
However if this is convention elsewhere, then that's fine, and I can change later.

Reply:
Gotcha. Was trying to fix it, but managed to break my rebase and am having A Time. I guess I could just slap a newline in in the editor.